cmd/core/pfcp_session_handlers.go

```diff
@@ -160,13 +178,34 @@ func HandlePfcpSessionDeletionRequest(conn *PfcpConnection, msg message.Message,
 			return message.NewSessionDeletionResponse(0, 0, 0, req.Sequence(), 0, newIeNodeID(conn.nodeId), ie.NewCause(ie.CauseRuleCreationModificationFailure)), err
 		}
 	}
+	for id := range session.URRs {
+		deletedURRs = append(deletedURRs, ie.NewUsageReportWithinSessionDeletionResponse(
+			ie.NewURRID(id),
+			// TODO: add getting actual values from ebpf.UrrInfo
+			ie.NewVolumeMeasurement(0x3f, 1, 0, 0, 0, 0, 0),
+		))
+		if err := mapOperations.DeleteUrr(id); err != nil {
+			PfcpMessageRxErrors.WithLabelValues(msg.MessageTypeName(), causeToString(ie.CauseRuleCreationModificationFailure)).Inc()
```

Comment:
It's better not to fail the whole command here. Just print warn log and go ahead.